.github/workflows/release.yml

```diff
@@ -62,8 +62,20 @@ env:
   DOCKER_REPOSITORY_PASSWORD: ${{ secrets.DOCKER_REPOSITORY_PASSWORD }}
   DOCKER_REPOSITORY_STORYBOOK: "${{ secrets.DOCKER_REPOSITORY_DOMAIN }}/alfresco/storybook"
   DOCKER_REPOSITORY: "${{ secrets.DOCKER_REPOSITORY_DOMAIN }}/alfresco/demo-shell"
-  NPM_REGISTRY_ADDRESS: ${{ secrets.NPM_REGISTRY_ADDRESS }}
-  NPM_REGISTRY_TOKEN: ${{ secrets.NPM_REGISTRY_TOKEN }}
+  REPO_OWNER: "Alfresco"
```

Comment:
there is no way to share those env instead of duplicating across workflows ?

Reply:
new task to make this action on env variables:
https://alfresco.atlassian.net/browse/HXOR-128